testing/fatutf8/fatutf8_main.c

```diff
@@ -52,6 +52,7 @@ static void show_usage(FAR const char *progname, int exitcode)
 #define BASEPATH "/mnt/sdcard"
 #define FOLDER_NAME "/ÄÖÜ 测试文件夹 Test Folder"
 #define FILE_NAME "/ÄÖÜ 测试文件 Test File"
+#define PATH_MAX 256
```

Comment:
why define PATH_MAX which is a standard macro?

Reply:
removed